src/absil/ilread.fs

```diff
@@ -1543,11 +1544,22 @@ let readBlobHeapAsDouble ctxt vidx = fst (sigptrGetDouble (readBlobHeap ctxt vid
 //        (e) the start of the native resources attached to the binary if any
 // ----------------------------------------------------------------------*)
 
-let readNativeResources (pectxt: PEReader)  = 
+let readNativeResources (pectxt: PEReader) = 
     [ if pectxt.nativeResourcesSize <> 0x0  && pectxt.nativeResourcesAddr <> 0x0 then 
-           let start = pectxt.anyV2P (pectxt.fileName + ": native resources", pectxt.nativeResourcesAddr)
-           yield ILNativeResource.In (pectxt.fileName, pectxt.nativeResourcesAddr, start, pectxt.nativeResourcesSize ) ]
-   
+        let start = pectxt.anyV2P (pectxt.fileName + ": native resources", pectxt.nativeResourcesAddr)
+        if pectxt.fromByteArray then
```

Comment:
Please add a comment that "fromBytes" is true only when "OpenILModuleReaderFromBytes" is used, i.e. when the DLL need not exist on disk, and that in this case we need to read the native resources eagerly.  

Please also rename `fromBytes` to `noFileOnDisk` or the like (at first I thought it was referring to when we use a `ByteFile` or other non-file access to the metadata section of the file).